l10n_it_vat_registries/__manifest__.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2011-2013 Associazione OpenERP Italia
+# (<http://www.openerp-italia.org>).
+# Copyright 2012 Domsense srl (<http://www.domsense.com>)
+# Copyright 2012-2018 Lorenzo Battistini - Agile Business Group
+# Copyright 2012-15 LinkIt srl (<http://http://www.linkgroup.it>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Italian Localization - VAT Registries',
```

Comment:
Dato che 'name' e 'summary' non sono traducibili in OCA meglio metterli direttamente in italiano:
Italian Localization - Registri IVA

Reply:
fatto

Reply:
Forse devi rigenerare il README.rst.